torchtune/__init__.py

```diff
@@ -12,13 +12,15 @@
     "alpaca_generate.py",
     "lora_finetune_single_device.py",
     "lora_finetune_distributed.py",
+    "eleuther_eval.py",
 ]
 _CONFIG_LISTS = {
     "full_finetune_single_device.py": ["full_finetune_single_device.yaml"],
     "full_finetune_distributed.py": ["full_finetune_distributed.yaml"],
     "lora_finetune_single_device.py": ["lora_finetune_single_device.yaml"],
     "lora_finetune_distributed.py": ["lora_finetune_distributed.yaml"],
     "alpaca_generate.py": ["alpaca_generate.yaml"],
+    "eleuther_eval.py": ["llama2_eleuther_eval.yaml"],
```

Comment:
Given we scrapped all the prefixes in our FT recipes, maybe just `eleuther_eval.yaml`? Thoughts?